account-and-billing/account-and-billing.helper.ts

```diff
@@ -0,0 +1,172 @@
+import { CoreConstants } from '../../../components/html/core-constants';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400120)
```suggestion
import { CoreConstants } from "../../../components/html/core-constants";
```